src/js/components/RangeSelector/RangeSelector.js

```diff
@@ -118,141 +124,124 @@ class RangeSelector extends Component {
-              ? // preserve existing dark, instead of using darknes
-                // of this color
-                {
-                  color: color || theme.rangeSelector.background.invert.color,
-                  opacity,
-                  dark: theme.dark,
-                }
-              : undefined
-          }
-          fill={fill}
-          round={round}
-        />
-        <EdgeControl
-          a11yTitle={messages.lower}
-          tabIndex={0}
-          ref={forwardRef}
-          color={color}
-          direction={direction}
-          edge="lower"
-          onMouseDown={onChange ? this.lowerMouseDown : undefined}
-          onDecrease={
-            onChange && lower - step >= min
-              ? () => onChange([lower - step, upper])
-              : undefined
-          }
-          onIncrease={
-            onChange && lower + step <= upper
-              ? () => onChange([lower + step, upper])
-              : undefined
-          }
-        />
-        <Box
-          style={{
-            flex: `${upper - lower + 1} 0 0`,
-            cursor: direction === 'vertical' ? 'ns-resize' : 'ew-resize',
-          }}
-          background={
-            invert
-              ? undefined
-              : // preserve existing dark, instead of using darknes of
-                // this color
-                { color: color || 'control', opacity, dark: theme.dark }
-          }
-          fill={fill}
-          round={round}
-          onMouseDown={onChange ? this.selectionMouseDown : undefined}
-        />
-        <EdgeControl
-          a11yTitle={messages.upper}
-          tabIndex={0}
-          color={color}
-          direction={direction}
-          edge="upper"
-          onMouseDown={onChange ? this.upperMouseDown : undefined}
-          onDecrease={
-            onChange && upper - step >= lower
-              ? () => onChange([lower, upper - step])
-              : undefined
-          }
-          onIncrease={
-            onChange && upper + step <= max
-              ? () => onChange([lower, upper + step])
-              : undefined
-          }
-        />
-        <Box
-          style={{ flex: `${max - upper} 0 0` }}
-          background={
-            invert
-              ? // preserve existing dark, instead of using darknes of this
-                // color
-                {
-                  color: color || theme.rangeSelector.background.invert.color,
-                  opacity,
-                  dark: theme.dark,
-                }
-              : undefined
-          }
-          fill={fill}
-          round={round}
-        />
-      </Container>
-    );
-  }
-}
+        round={round}
+      />
+      <EdgeControl
+        a11yTitle={messages.lower}
+        tabIndex={0}
+        ref={forwardRef}
+        color={color}
+        direction={direction}
+        edge="lower"
+        onMouseDown={onChange ? lowerMouseDown : undefined}
+        onDecrease={
+          onChange && lower - step >= min
```

Comment:
This `lower - step >= min` could be put in a variable to make it clearer of what it is exactly, what do you think?